pages/index.tsx

```diff
@@ -86,14 +103,14 @@ const Home: NextPage = (props) => {
       </Head>
 
       <main className="flex flex-col items-center justify-center flex-1 w-full px-20 text-center bg-slate-500">
-        <DragDropContext onDragEnd={onDragEnd}>
+        {winReady ? <DragDropContext onDragEnd={onDragEnd}>
```

Comment:
only renders DragDropContext when winReady is set

Reply:
it works for me, but what is the condition "else"?
is it null?